### PR TITLE
add reusable workflow to generate prebuilds

### PR DESCRIPTION
**Pull request by rochdev**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @DataDog/apm-js
```
Comment (Qard) [inline CODEOWNERS:1]: GitHub seems to be complaining about this. I guess this team needs write access?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+```yaml
+jobs:
+  build:
+    uses: Datadog/action-prebuildify/.github/workflows/build.yml@v1
+    with:
+      target-name: 'addon' # target name in binding.gyp
+      package-manager: 'npm' # npm or yarn
+      cache: false # enable caching of dependencies based on lockfile
+```
```
Comment (Qard) [inline README.md:23]: Unclear from the description, but this is also able to run the tests on each platform too? Or just make the prebuilds?

Comment (rochdev) [reply README.md:23]: It runs `npm test` on the parent workflow.

**Review by Qard — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (rochdev) [reply CODEOWNERS:1]: Fixed.

**Review by bengl — APPROVED** [review]

**Pull request merged**